docs/store/change-metrics-storage.md

```diff
@@ -51,10 +51,9 @@ the accuracy of the values you enter below, changes in the compression ratio, an
 
 :::
 
-Download
-the [calculator](https://docs.google.com/spreadsheets/d/e/2PACX-1vTYMhUU90aOnIQ7qF6iIk6tXps57wmY9lxS6qDXznNJrzCKMDzxU3zkgh8Uv0xj_XqwFl3U6aHDZ6ag/pub?output=xlsx)
-to optimize the data retention to your preferences. Utilize the "Front" spreadsheet. Experiment with the variables which
-are padded with yellow to come up with the best settings for your use case.
+Visit our [Google Sheet calculator](https://docs.google.com/spreadsheets/d/1aRLZJGJk6fvQDktkCB44l-EFROJcUrwOLe8rBQcfpPE/edit?usp=sharing)
+to tune the data retention on your preferences. Experiment with the variables which are padded with yellow 
```

Comment:
should we make it explicit that you need to "Make a copy" to tune the data retention on your preferences?